libsnark/zk_proof_systems/ppzksnark/r1cs_gg_ppzksnark/r1cs_gg_ppzksnark.hpp

```diff
@@ -169,38 +169,35 @@ class r1cs_gg_ppzksnark_verification_key {
 public:
```

Comment:
I'd add https://github.com/zcash/sapling-security-analysis/blob/master/secondmpc.pdf as a reference, and probably a few comments about to explicit the structure of the SRS (VK, PK)